Aquality.Appium.Mobile/src/Aquality.Appium.Mobile/Elements/Actions/ElementTouchActions.cs

```diff
@@ -0,0 +1,101 @@
+﻿using Aquality.Appium.Mobile.Actions;
+using Aquality.Appium.Mobile.Applications;
+using Aquality.Appium.Mobile.Configurations;
+using Aquality.Selenium.Core.Elements.Interfaces;
+using System.ComponentModel;
+using System.Drawing;
+
+namespace Aquality.Appium.Mobile.Elements.Actions
+{
+    public class ElementTouchActions : IElementTouchActions
+    {
+        private readonly IElement element;
+        private readonly Point scrollDownStartPoint;
+        private readonly Point scrollDownEndPoint;
+        private readonly Point swipeLeftStartPoint;
+        private readonly Point swipeLeftEndPoint;
+        private readonly Point scrollUpStartPoint;
+        private readonly Point scrollUpEndPoint;
+        private readonly Point swipeRightStartPoint;
+        private readonly Point swipeRightEndPoint;
+        private readonly ITouchActionsSettings touchActionsSettings = AqualityServices.Get<ITouchActionsSettings>();
+
+        public ElementTouchActions(IElement element)
+        {
+            this.element = element;
+            scrollDownStartPoint = RecalculatePointCoordinates(
+                GetBottomRightCornerPoint(),
+                1 - touchActionsSettings.SwipeHorizontalOffset,
```

Comment:
from my sense of code style, I would prefer to store touchActionsSettings.SwipeHorizontalOffset and touchActionsSettings.SwipeVerticalOffset to variables rather than reference them from settings. These values are not initialized in TouchActionsSettings' ctor, so they will be reinitialized each time we reference them.